conduct.md

```diff
@@ -88,13 +90,15 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
+- Always be transparent on what are we doing wrong with. The reason being is because to create a mutual relationship environment between the team to understand the good and the bad of the activity together.
```

Comment:
Please rephrase the first sentence -- maybe something more like 'Be transparent about any technical or personal problems, issues, and conflicts they experience to their team members, TLs, and Professor.' Additionally, you want to rephrase the second sentence to, maybe something like 'This will create a mutually beneficial environment between team members, where they are comfortable sharing the difficulties and positive experiences of their group activities.'

Reply:
Was written by another team that already merged to main--content was pulled in when branch was created